cloudx/handler.go

```diff
@@ -231,6 +231,30 @@ func (h *SnakeCharmer) EnsureContext() (*AuthContext, error) {
 	}
 
 	if len(c.SessionToken) > 0 {
+		client, err := newKratosClient()
+		if err != nil {
+			return nil, err
+		}
+		_, resp, err := client.V0alpha2Api.ToSession(h.ctx).XSessionToken(c.SessionToken).Execute()
+		if resp.StatusCode == http.StatusUnauthorized || err != nil {
+			if h.isQuiet {
+				return nil, errors.New("can not sign in or sign up when flag --quiet is set")
```

Comment:
Can you please add an error message that explicitly states that the session has expired? Please also add a test for this behaviour.